tools/cli-bootstrap/dockerfiles/debian8/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:8.0
+MAINTAINER Bryan P. Arant <bryanar@microsoft.com>
+RUN apt-get -qqy update
```

Comment:
Consider doing all this setup in one run command to reduce the number of layers that are constructed.  Also consider cleaning the apt-get cache as part of the same command to reduce layer size (the dockerfiles that we have for dotnet core should do this if you need an example).